.gitmodules

```diff
@@ -70,3 +70,9 @@
 [submodule "vendor/github.com/pkg/profile"]
 	path = vendor/github.com/pkg/profile
 	url = https://github.com/pkg/profile
+[submodule "vendor/github.com/bboreham/arping"]
```

Comment:
Your [PR](https://github.com/j-keck/arping/pull/1) has been merged, so I assume that this submodule is not  needed.